frontend/src/components/suggestions/SuggestionsList.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC, useState } from 'react';
+import { Suggestion } from './types';
+import { SuggestionItem } from './SuggestionItem';
+import { Typography, Box, Checkbox, Button, makeStyles, createStyles, Theme } from '@material-ui/core';
+import { Playlist } from '../playlists/types';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    actions: {
+      '& button': {
+        marginLeft: theme.spacing(2)
+      }
+    }
+  })
+);
+
+interface SuggestionsListProps {
+  suggestions: Suggestion[];
+  playlist?: Playlist;
+  onSuggestionsAdded?: (playlist: Playlist, suggestions: Suggestion[]) => void;
+}
+export const SuggestionsList: FC<SuggestionsListProps> = ({ suggestions, playlist, onSuggestionsAdded }) => {
+  const classes = useStyles();
+
+  const initialState = suggestions.reduce(
+    (acc, suggestion, idx) => {
+      acc[idx] = { suggestion, selected: false };
```

Comment:
`acc` ?

Reply:
In a reduce function the 1st parameter is the accumulator you hold until the end of the iteration, so `acc` stands for `accumulator`